activerecord/lib/active_record/connection_adapters/schema_cache.rb

```diff
@@ -196,10 +192,6 @@ def indexes(table_name)
         @schema_reflection.indexes(@connection, table_name)
       end
 
-      def database_version
```

Comment:
If we don't backport this PR, we should sneak a `:nodoc:` in here on 7-1-stable. The existing SchemaCache method already was (https://github.com/rails/rails/blob/7-0-stable/activerecord/lib/active_record/connection_adapters/schema_cache.rb#L144), but it looks like I failed to propagate that up to here + SchemaReflection.

Reply:
Good catch.

Reply:
Done in https://github.com/rails/rails/commit/5a201b5b44ff160d800e9aab04c462abf7e27569.

As for backporting this to `7-1-stable`, I don't think it's necessary.